tensorflow/compiler/tf2xla/kernels/gpu_tf_kernel_custom_call.cc

```diff
@@ -504,10 +504,13 @@ static void GenericTfCallback(se::gpu::GpuStreamHandle stream_handle,
                                   s.error_message().size());
   }
 }
-
+#if GOOGLE_CUDA
 XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM(kTfCallbackCustomCall,
                                          GenericTfCallback, "CUDA");
-
+#else
+XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM(kTfCallbackCustomCall,
+                                         GenericTfCallback, "ROCM");
+#endif
 }  // namespace
```

Comment:
I have a PR up which has this change plus some other stuff:

https://github.com/tensorflow/tensorflow/pull/56342

Don't remove this change here because my PR is on the upstream repo only.

Just FYI.